TypeCobol/Compiler/Scanner/Scanner.cs

```diff
@@ -1141,12 +1145,16 @@ public Token GetNextToken()
                             else
                             {
                                 followingChar = line[currentIndex];
+                                followingChar2 = (currentIndex + 1) < line.Length ? line[currentIndex + 1]: '\0';
                             }
 
                             delimiterToken = new Token(TokenType.PseudoTextDelimiter, startIndex, startIndex + 1, usesVirtualSpaceAtEndOfLine, tokensLine);
                             if (!(followingChar == ' ' || followingChar == ',' || followingChar == ';' || followingChar == '.'))
                             {
-                                tokensLine.AddDiagnostic(MessageCode.InvalidCharAfterPseudoTextDelimiter, delimiterToken);
+                                if (followingChar != '=' || followingChar2 != '=')
+                                {//If not an empty placeholder this is an error.
+                                    tokensLine.AddDiagnostic(MessageCode.InvalidCharAfterPseudoTextDelimiter, delimiterToken);
+                                }
```

Comment:
This is necessary here in case of ====

Reply:
Do we have test for this?

Reply:
Yes PseudoText.txt and TestTokenTypes.CheckPseudoText();

Reply:
I don't see any changes in this test in this PR.
Can you test for the new condition ?

Reply:
As I understand the code, we are reading the two following chars after the _closing_ pseudo-text delimiter so this is wrong. Let's consider this code

```cobol
COPY YAAACPY REPLACING ==== BY ====.
```

For the first pseudo-text, `followingChar` would be ` ` (space) and followingChar2 would be `B` (from the BY keyword) and for the second pseudo-text we would get `.` for followingChar and probably `\n` for followingChar2 (I haven't tested)

Reply:
The new condition is for keeping the diagnostics in PseudoText.txt and TestTokenTypes.CheckPseudoText() tests, otherwise the diagnostics will not be generated. This test is not for the BY but for the condition _if (tokensLine.ScanState.InsidePseudoText)_
thus it is applyed inside a placeholder.

Reply:
Ok, I made a mistkake using smedilol method this if is useless.